ext/intl/php_intl.stub.php

```diff
@@ -330,7 +330,7 @@ function msgfmt_format_message(string $locale, string $pattern, array $values):
 
 function msgfmt_parse(MessageFormatter $formatter, string $string): array|false {}
 
-function msgfmt_parse_message(string $locale, string $pattern, string $message): array|false {}
+function msgfmt_parse_message(string $locale, string $pattern, string $message): array|false|null {}
```

Comment:
Can you explain under which circumstances this one can return null? I couldn't spot it.

Reply:
Nor could I spot it now. So I'll prepare a fix for this as well

Reply:
I've dropped it in https://github.com/php/php-src/commit/9fe4966811e75c33a403bc179a55acbeaa4ffe66.